api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -404,7 +404,7 @@ public DbUser setComputeSecuritySuspendedUntil(Timestamp computeSecuritySuspende
   }
 
   @OneToOne(
-      cascade = CascadeType.ALL,
```

Comment:
https://stackoverflow.com/questions/13370221/persistentobjectexception-detached-entity-passed-to-persist-thrown-by-jpa-and-h